src/components/collapsible/CollapsibleActions.ts

```diff
@@ -0,0 +1,51 @@
+import {IReduxAction} from '../../utils/ReduxUtils';
+
+export const CollapsibleActions = {
+    add: 'ADD_COLLAPSIBLE',
+    remove: 'REMOVE_COLLAPSIBLE',
+    setExpanded: 'SET_COLLAPSIBLE_EXPANDED',
+};
+
+export interface ILoadingActionPayload {
+    id: string;
+    expanded?: boolean;
+}
+
+/**
+ * @deprecated use addCollapsible instead
+ */
+export const addCollapsibleContainer = (id: string, expanded: boolean): IReduxAction<ILoadingActionPayload> => ({
+    type: CollapsibleActions.add,
+    payload: {id, expanded},
+});

```

Comment:
You could call `addCollapsible(id, expanded)` instead of repeating the whole action. (same for other actions)